auto/sources

```diff
@@ -108,7 +109,10 @@ NXT_LIB_SRC0=" \
     src/nxt_mem_pool_cleanup.c \
 "
 
-NXT_LIB_UNIT_SRCS="src/nxt_unit.c"
```

Comment:
No need to add `src/nxt_socket_msg.c` here. It is already in `NXT_LIB_SRCS`. Please revert this change.